server/database/queries/selectUser.js

```diff
@@ -0,0 +1,3 @@
+const connection = require('../config/connection');
+
+module.exports = id => connection.query('select first_name, last_name, email, avatar, organisation_name, business_type, website, address, city, country, zip_code, facebook, instagram, twitter from "user" where id = $1', [id]);
```

Comment:
you can select everything from database and then in your controller you can remove the password 
```suggestion
module.exports = id => connection.query('select * from "user" where id = $1', [id]);
```